iree_tests/onnx/import_tests.py

```diff
@@ -131,8 +132,8 @@ def import_onnx_files(test_dir_path, imported_dir_path):
 
     # Convert input_*.pb and output_*.pb to .npy files.
```

Comment:
Please update the comment here (using binary .bin files instead of .npy files)

Could also add a note about endianness if we want to be really clear (the binary format won't work across systems with different endianness IIUC)

Reply:
That's right, I did come across the endianness aspect of using struct.packing, added the comment for big endian systems to import tests locally to have big endian bins. Thanks for noticing this!